milli/src/update/index_documents/transform.rs

```diff
@@ -340,35 +340,45 @@ impl<'a, 'i> Transform<'a, 'i> {
             return Ok(None);
         }
 
+        // store the keys and values of the json + the original obkv
```

Comment:
I am not sure to understand this comment, where are the keys, values, and original obkv in this vec of tuples?

Reply:
In fact the right part of the tuple contains the original obkv value which is a serialized JSON, right?

Reply:
The values are either coming from the obkv or from the flattened json, depending on if it was necessary to flatten any fields.

Reply:
Ho, Could you specify that in the comment please? Just to make it more clear :)

Reply:
Done :+1: